library/core/src/ops/arith.rs

```diff
@@ -462,6 +462,10 @@ mul_impl! { usize u8 u16 u32 u64 u128 isize i8 i16 i32 i64 i128 f32 f64 }
 #[lang = "div"]
 #[stable(feature = "rust1", since = "1.0.0")]
 #[rustc_on_unimplemented(
+    on(
+        all(any(_Self = "f32", _Self = "f64",), Rhs = "{integer}",),
+        note = "consider using a floating-point literal by writing it with `.0`"
```

Comment:
I don't actually know: should this be generated in the code, instead of via the attribute, now?

Reply:
I agree with you. I fixed it as you said.